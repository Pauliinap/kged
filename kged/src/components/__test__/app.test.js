import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import App from '../app'

it('app is rendered with expected structure', () => {
    const app = shallow(<App />);
    expect(app).toMatchSnapshot();
})
