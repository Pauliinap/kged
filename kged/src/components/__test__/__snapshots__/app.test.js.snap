// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app is rendered with expected structure 1`] = `
<div
  className="app container-fluid"
>
  <div
    className="row flex-grow-1"
  >
    <Sidebar />
    <div
      className="col-lg-7 order-last pre-container"
    >
      <Connect(TitleBar) />
      <Connect(ActionBar) />
      <Connect(Preview) />
      <Connect(Console) />
    </div>
    <Connect(Inspector) />
  </div>
</div>
`;
